src/plays/tic-tac-toe/Icon.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { FaTimes, FaRegCircle } from "react-icons/fa";
+
+const Icon = ({ choice }) => {
+  switch (choice) {
+    case "cross":
+      return <FaTimes className="icon" />;
+    case "circle":
+      return <FaRegCircle className="icon" />;
+    default:
+      return "";

```

Comment:
Let's maybe return null or undefined as a default case, cuz from Icon you expect to receive a ReactElement or nothing, but not an empty string